plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator_test.go

```diff
@@ -458,3 +459,92 @@ valuesMerge: replace
 			111,             // memory
 		))
 }
+
+func TestHelmChartInflationGeneratorWithIncludeCRDs(t *testing.T) {
+	th := kusttest_test.MakeEnhancedHarnessWithTmpRoot(t).
+		PrepBuiltin("HelmChartInflationGenerator")
+	defer th.Reset()
+	if err := th.ErrIfNoHelm(); err != nil {
+		t.Skip("skipping: " + err.Error())
+	}
+
+	// we store this data outside of the _test.go file as its sort of huge
+	// and has backticks, which makes string literals wonky
+	testData, err := ioutil.ReadFile("include_crds_testdata.txt")
+	if err != nil {
+		t.Errorf("unable to read test data for includeCRDs: %w", err)
+	}
+
+	rm := th.LoadAndRunGenerator(`
+apiVersion: builtin
+kind: HelmChartInflationGenerator
+metadata:
+  name: terraform
+name: terraform
+version: 1.0.0
+repo: https://helm.releases.hashicorp.com
```

Comment:
I decided to pick another chart here, as it conveniently let me toggle off every template and only include CRDs. :)

Reply:
nice